aten/src/ATen/native/cuda/Col2Im.cu

```diff
@@ -119,28 +116,26 @@ void col2im_out_cuda_template(
             stride_width +
         1;
 
-    for (int64_t elt = 0; elt < batch_size; elt++) {
-      input_n = input.select(0, elt);
-      output_n = output.select(0, elt);
-
-      col2im<scalar_t, accscalar_t>(
```

Comment:
is col2im/col2im_kernel still used anywhere, or can they be removed?

Reply:
Yes, It's used here for example:
https://github.com/pytorch/pytorch/blob/fa99b7b8f726f8ce63f0dff076b7e9171e3dd40a/aten/src/ATen/native/cuda/ConvolutionMM2d.cu#L281